packages/ember-glimmer/lib/components/curly-component.js

```diff
@@ -32,6 +41,30 @@ function attrsToProps(keys, attrs) {
   return merged;
 }
 
+function keysForAttrs(args, posParamsDefinition) {
+  let namedKeys = args.named.keys;
+  let posKeys = keysForPositionalParameters(args, posParamsDefinition);
+
+  return namedKeys.concat(posKeys);
+}
+
+function keysForPositionalParameters(args, posParamsDefinition) {
+  let numberOfPosParams = args.positional.length;
+  if (numberOfPosParams === 0) {
+    // If there are no positional parameters passed, returned an empty array
+    return [];
```

Comment:
`args.positional` are the values of the positional parameters but we are looking for an array of keys to pass to `attrsToProps`.